### PR TITLE
Added two papers

### PR DESCRIPTION
**Pull request by ChopinSharp**
Added two papers regarding referring expression segmentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +137,8 @@ ConvNet (SCRC)* [[Paper]](https://www.cv-foundation.org/openaccess/content_cvpr_
 
 1. Margffoy-Tuay, Edgar, et al. **Dynamic multimodal instance segmentation guided by natural language queries.** Proceedings of the European Conference on Computer Vision (ECCV). 2018. [[Paper]](https://arxiv.org/pdf/1807.02257.pdf) [[Code]](https://github.com/BCV-Uniandes/DMS)
 
+1. Shi, Hengcan, et al. **Key-word-aware network for referring expression image segmentation.** Proceedings of the European Conference on Computer Vision (ECCV). 2018.[[http://openaccess.thecvf.com/content_ECCV_2018/papers/Hengcan_Shi_Key-Word-Aware_Network_for_ECCV_2018_paper.pdf
```
Comment (TheShadow29) [inline README.md]: Change it to ```[[Paper]](paper_link)```. Also, it seems they have a github repo (https://github.com/shihengcan/key-word-aware-network-pycaffe). You could add that using ```[[Code]](code_link)```

File: README.md [hunk 2]
```diff
@@ -125,6 +125,8 @@ ConvNet (SCRC)* [[Paper]](https://www.cv-foundation.org/openaccess/content_cvpr_
 
 1. Deng, Chaorui, et al. **Visual Grounding via Accumulated Attention.** Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition. 2018.[[Paper]](http://openaccess.thecvf.com/content_cvpr_2018/papers/Deng_Visual_Grounding_via_CVPR_2018_paper.pdf)
 
+1. Li, Ruiyu, et al. **Referring image segmentation via recurrent refinement networks.** Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition. 2018.[[Paper]](http://openaccess.thecvf.com/content_cvpr_2018/papers/Li_Referring_Image_Segmentation_CVPR_2018_paper.pdf)[[code]](https://github.com/liruiyu/referseg_rrn)
```
Comment (TheShadow29) [inline README.md]: For consistency please capitalize `code` to `Code`

**Review by TheShadow29 — CHANGES_REQUESTED** [review]
Thank you for your interest. The papers seem very relevant. Could you make the small changes I suggested, after that I can merge the pull request.

**Comment by ChopinSharp** [comment]
> Thank you for your interest. The papers seem very relevant. Could you make the small changes I suggested, after that I can merge the pull request.

Sorry for the late reply. I think i've made the changes as you requested. Please verify.

**Comment by TheShadow29** [comment]
Great! Thanks for your contribution! I have merged the changes.

**Pull request merged**